product_docs/docs/efm/4/efm_user/04_configuring_efm/01_cluster_properties/index.mdx

```diff
@@ -576,7 +576,9 @@ To perform maintenance on the primary database when `primary.shutdown.as.failure
 
 <div id="update_physical_slots_period" class="registered_link"></div>
 
-Use the `update.physical.slots.period` property to define the slot advance frequency for database version 12 and above. When `update.physical.slots.period` is set to a non-zero value, the primary agent will read the current `restart_lsn` of the physical replication slots after every `update.physical.slots.period` seconds, and send this information with its `pg_current_wal_lsn` and `primary_slot_name` (If it is set in the postgresql.conf file) to the standbys. If physical slots do not already exist, setting this parameter to a non-zero value will create the slots and then update the `restart_lsn parameter` for these slots. A non-promotable standby will not create new slots but will update them if they exist.
+Use the `update.physical.slots.period` property to define the slot advance frequency for database version 12 and above. When `update.physical.slots.period` is set to a non-zero value, the primary agent will read the current `restart_lsn` of the physical replication slots after every `update.physical.slots.period` seconds, and send this information with its `pg_current_wal_lsn` and `primary_slot_name` (if it is set in the postgresql.conf file) to the standbys. If physical slots do not already exist, setting this parameter to a non-zero value will create the slots and then update the `restart_lsn parameter` for these slots. A non-promotable standby will not create new slots but will update them if they exist.
```

Comment:
Instead of saying `non-zero` can we say something like `greater-than-zero` or `positive integer` ? A value of `-1` would still be `non-zero` but an invalid value

Reply:
@Jagdish-Kewat-EDB I don't know how to make more edits to an existing PR without access to the repo. You can create a new one if you think it's worth it.

Reply:
There are a few ways you can edit a PR from GitHub including:

- Select + next to the line you want to edit, select the first icon (insert a suggestion), make the change and commit it
- Select Edit from the ... menu, edit the file, and commit your change.

I can do a quick screen share with you if you like.

Reply:
I can also make the change for you.

Reply:
@drothery-edb Great!